README.md

```diff
@@ -4,10 +4,10 @@
 
 Setup | Status
 --- | ---
-Minimal | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-core%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-core/job/master/)
-ZMQ | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-zmq-core%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-zmq-core/job/master/)
-ROS 16.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-16.04%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-16.04/job/master/)
-ROS 18.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-18.04%2Fmaster)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-18.04/job/master/)
+Minimal | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-core%2Frm-zhelpers-deps)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-core/job/rm-zhelpers-deps/)
+ZMQ | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-zmq-core%2Frm-zhelpers-deps)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-zmq-core/job/rm-zhelpers-deps/)
+ROS 16.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-16.04%2Frm-zhelpers-deps)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-16.04/job/rm-zhelpers-deps/)
+ROS 18.04 | [![Build Status](https://jenkins1.klepsydra.com/buildStatus/icon?job=kpsr-ros-core-18.04%2Frm-zhelpers-deps)](https://jenkins1.klepsydra.com/view/Core/job/kpsr-ros-core-18.04/job/rm-zhelpers-deps/)
 
```

Comment:
Is there any way to change this strings to master and hence avoid turning them to branch name? It happens when commiting via CLI or VScode GUI